compute/oslogin/service_account_ssh_test.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2019, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import random
+import base64
+import json
+import googleapiclient.discovery
+from google.oauth2 import service_account
+from service_account_ssh import main
+
+'''
+The service account that runs this test must have the following roles:
+- roles/compute.instanceAdmin.v1
+- roles/compute.securityAdmin
+- roles/iam.serviceAccountAdmin
+- roles/iam.serviceAccountKeyAdmin
+- roles/iam.serviceAccountUser
+
+The Project Editor legacy role is not sufficient because it does not grant
+several necessary permissions.
+'''
+
+def test_main(capsys):
```

Comment:
Lint wants two blank lines here, not just one.

Reply:
Done.